conanfile.py

```diff
@@ -163,8 +163,10 @@ def _conan_installs(self):
     def _python_installs(self):
         python_installs = {}
 
+        python_executable = "python3" if self.settings.os == "Macos" else "python"
+
         # list of python installs
-        python_ins_cmd = f"python -c \"import pkg_resources; print(';'.join([(s.key+','+ s.version) for s in pkg_resources.working_set]))\""
+        python_ins_cmd = f"{python_executable} -c \"import pkg_resources; print(';'.join([(s.key+','+ s.version) for s in pkg_resources.working_set]))\""
```

Comment:
This shouldn't be needed anymore with my recent changes. If it is still needed then apply the environment context to this, see:

- https://github.com/Ultimaker/conan-config/pull/10
- https://github.com/Ultimaker/conan-config/pull/9
- https://github.com/Ultimaker/Cura/pull/18138